htdocs/css/panel.css

```diff
@@ -46,19 +46,19 @@ Author: Evan McIlroy <evanmcilroy@gmail.com>*/
 .navbar{
     background-color: #064785;
     border-width: 0 0 1px;
-    border-radius: 0px;
+    border-radius: 0;
 }
 
 .navbar-toggle{
-	margin-top: 0px;
-	margin-bottom: 0px;
-	margin-right: 0px;
-	border: 0px;
-	border-radius: 0px;
+	margin-top: 0;
+	margin-bottom: 0;
+	margin-right: 0;
+	border: 0;
+	border-radius: 0;
```

Comment:
Some more inconsistent indentation.

Reply:
Discussed on slack conclusion "it's better" to have the space between the colon & value. 
The 4 spaces for indentation seems better for reading to me. Latest commit addressed making it consistent.

Reply:
@intralizee but this section has 4 spaces indentation, and the sections around it only has 2